rclpy/rclpy/node.py

```diff
@@ -1949,3 +1963,31 @@ def get_subscriptions_info_by_topic(
             topic_name,
             no_mangle,
             _rclpy.rclpy_get_subscriptions_info_by_topic)
+
+    def wait_for_node(
+        self,
+        node_name: str,
```

Comment:
nit:

I would use:
```suggestion
        fq_node_name: str,
```
or `fully_qualified_node_name`, to make it more explicit.

Reply:
Done: [90e8bf0](https://github.com/ros2/rclpy/pull/930/commits/90e8bf0f56903c40ee4a6741e85acbb5d5cedb0f)